File: CSharp/1893.cs
using System;
using System.Linq;
class Program
{
    static void Main(string[] args)
    {
        int[] lua = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int a = lua[0];
        int b = lua[1];

        if(b>=0 && b<=2)
        {
            Console.WriteLine("nova");
        }
        else if(b>=3 && b<=96 & b>a)
        {
            Console.WriteLine("crescente");
        }
        else if(b>=97 && b<=100)
        {
            Console.WriteLine("cheia");
        }
        else if(b>=3 && b<=96 & b<a)
        {
            Console.WriteLine("minguante");
        }

    }
}

File: Python/1893.py
lua = list(map(int, input().split()))
a = lua[0]
b = lua[1]

if 0 <= b <= 2:
    print("nova")
elif 3 <= b <= 96 and b > a:
    print("crescente")
elif 97 <= b <= 100:
    print("cheia")
elif 3 <= b <= 96 and b < a:
    print("minguante")
